liquibase-standard/src/main/java/liquibase/precondition/core/SequenceExistsPrecondition.java

```diff
@@ -1,21 +1,28 @@
 package liquibase.precondition.core;
 
 import liquibase.changelog.ChangeSet;
-import liquibase.changelog.visitor.ChangeExecListener;
 import liquibase.changelog.DatabaseChangeLog;
+import liquibase.changelog.visitor.ChangeExecListener;
 import liquibase.database.Database;
+import liquibase.database.core.PostgresDatabase;
+import liquibase.database.jvm.JdbcConnection;
 import liquibase.exception.*;
 import liquibase.precondition.AbstractPrecondition;
-import liquibase.snapshot.DatabaseSnapshot;
 import liquibase.snapshot.SnapshotGeneratorFactory;
 import liquibase.structure.core.Schema;
 import liquibase.structure.core.Sequence;
 
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
 public class SequenceExistsPrecondition extends AbstractPrecondition {
     private String catalogName;
     private String schemaName;
     private String sequenceName;
 
+    private static final String POSTGRESQL_SQL_CHECK = "SELECT c.relname FROM pg_class c JOIN pg_namespace ns on c.relnamespace = ns.oid WHERE c.relkind = 'S' AND ns.nspname = ? and c.relname = ?";
```

Comment:
Can we add a comment in the code as to this query's purpose, or maybe update its name (i.e. `POSTGRES_EXISTENCE_SEQUENCE_SQL_CHECK`)?